src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpContext.java

```diff
@@ -42,6 +41,9 @@
  */
 public abstract class HttpContext {
 
+    /**
+     * Creates a HttpContext.
```

Comment:
I'd suggest to keep consistent... You have been using * Constructor for subclasses to call at other places. Maybe you should use that sentence here too.

Reply:
Replaced wording as suggested.